packages/schematics/angular/migrations/update-12/update-angular-config.ts

```diff
@@ -30,29 +30,39 @@ const ServerBuilderOptions: BuilderOptionsType = [
 ];
 
 export default function (): Rule {
-  return updateWorkspace((workspace) => {
-    for (const [, target] of allWorkspaceTargets(workspace)) {
-      if (!target?.builder.startsWith('@angular-devkit/build-angular')) {
-        continue;
-      }
+  return (_tree, context) =>
+    updateWorkspace((workspace) => {
+      for (const [targetName, target, projectName] of allWorkspaceTargets(workspace)) {
+        if (
+          !target.builder.startsWith('@angular-devkit/') &&
+          !target.builder.startsWith('@nguniversal/')
+        ) {
+          context.logger.warn(tags.stripIndent`
+            "${targetName}" target in "${projectName}" project is using a third-party builder.
+            You may need to adjust the options to retain the existing behavior.
+            For more information, see https://github.com/angular/angular-cli/releases/tag/v12.0.0
```

Comment:
```suggestion
            For more information, see the breaking changes section within the release notes: https://github.com/angular/angular-cli/releases/tag/v12.0.0
```

The release notes are quite long.  It would be nice if we could link directly to the breaking changes section.

Reply:
Yeah that would be nice, I tried using anchor tags but it doesn’t appear to work